physical/raft/snapshot_test.go

```diff
@@ -168,7 +168,7 @@ func TestRaft_Snapshot_Index(t *testing.T) {
 
 	// Get index
 	index, _ = raft.fsm.LatestState()
-	if index.Term != 1 {
+	if index.Term != 2 {
```

Comment:
How come the term is now 2?

Reply:
Change upstream in the raft library, we used to rely on a `StartAsLeader` flag that was removed. Now the voting takes place an increases the term number